msm-sitemap.php

```diff
@@ -699,8 +715,8 @@ public static function build_root_sitemap_xml( $year = false ) {
 	 */
 	public static function build_sitemap_url( $sitemap_date ) {
 		$sitemap_time = strtotime( $sitemap_date );
-		$post_type = get_query_var( 'type' );
-		$post_type = empty( $post_type ) ? '' : $post_type;
+		$post_type = get_query_var( 'post_type' );
```

Comment:
We should check if this post type is registered as well here using `post_type_exists`.